features/fields-fout-cases.feature

```diff
@@ -0,0 +1,140 @@
+#language: nl
+
+@post-assert
+Functionaliteit: Fields fout cases
+
+Rule: De Fields parameter is verplicht.
+
+  @fout-case
+  Scenario: Fields parameter ontbreekt
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999994086                       |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                                                                      |
+    | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.                                                   |

```

Comment:
@MelvLee is dit een parametercombinatiefout of een parametervalidatiefout?

Reply:
burgerservicenummer en fields zijn verplichte parameters. Als één of meerdere van de verplichte parameters niet voorkomen is het parametercombinatiefout. parametervalidatiefout slaat op de inhoud van de parameter (is mijn interpretatie uit de andere HC APIs)